attributes/default.rb

```diff
@@ -125,6 +125,7 @@
 default['hopsworks']['hive_default_quota_mbs']           = "250000"
 default['hopsworks']['featurestore_default_quota_mbs']   = "250000"
 default['hopsworks']['max_num_proj_per_user']            = "10"
+default['hopsworks']['reserved_project_names']           = "python27,python36,python37,python38,python39,hops-system,hopsworks,information_schema,airflow,glassfish_timers,grafana,hops,metastore,mysql,ndbinfo,performance_schema,sqoop,sys"
```

Comment:
@kouzant @SirOibaf shouldn't we add the os users as well?

Reply:
Why would we blacklist OS users?